treeherder/model/fixtures/repository.json

```diff
@@ -1280,11 +1280,11 @@
         "dvcs_type": "git",
         "name": "servo-try",
         "url": "https://github.com/servo/servo",
-        "branch": "try",
+        "branch": "try,try-taskcluster,try-linux,try-mac,try-windows,try-wpt,try-wpt,try-wpt,try-android,try-magicleap,try-arm",
```

Comment:
There are three try-wpt entries here.

Reply:
I think you want try-wpt-mac and try-wpt-android along with try-wpt?

Reply:
Good catch! I’m new at this multi-cursor text editing and should have looked more closely at what I copy/pasted…